assets/components/src/color-picker/style.scss

```diff
@@ -0,0 +1,164 @@
+/**
```

Comment:
I'm seeing the circle around selected color slightly off-center:
 
<img width="583" alt="Screen Shot 2020-02-08 at 6 25 50 PM" src="https://user-images.githubusercontent.com/1477002/74093600-27233e80-4aa2-11ea-929f-f51c15ace76a.png">

Reply:
Really strange one as this did not happen in the editor. Should be fixed in https://github.com/Automattic/newspack-plugin/pull/431/commits/f82101e4c23ca6353c29ebe8f3e7231e05e1b8bb